database.js

```diff
@@ -0,0 +1,20 @@
+const mysql = require('mysql');
+const util = require('util');
+
+let databasePool = mysql.createPool({
```

Comment:
umieść całą konfigurację w osobnym pliku konfiguracyjnym, a tutaj tylko zaczytaj konkretne wartości. Poczytaj o separate of concerns: oddzielamy konfigurację od implementacji